Applications/Games/Spore/Steam/script.js

```diff
@@ -0,0 +1,18 @@
+include(["engines", "wine", "quick_script", "steam_script"]);
+
+var installerImplementation = {
+    run: function () {
+        new SteamScript()
+            .name("Spore")
+            .editor("Maxis")
+            .author("Zemogiter")
+            .applicationHomepage("http://www.spore.com/")
+            .wineDistribution("upstream")
+            .wineVersion("4.0-rc1")
```

Comment:
Why different Wine version for both scripts?

Reply:
Because in wine 4.0-rc1 they fixed Steam Browser. But I haven't tested how origin behaves in that wine version.

Reply:
Update: Just tested Origin under Wine 4.0-rc1 and the update crashes it with error code 9:0. Reverting it back to 3.19-staging fixes that.